include/openssl/ocsp.h

```diff
@@ -54,26 +54,26 @@ extern "C" {
 // this excludes certificates request/response and ignores the |certs|
 // parameter. Certificates are optional.
 #define OCSP_NOCERTS 0x1
-// OCSP_NOINTERN is for |OCSP_basic_verify|. Certificates included within |bs|
-// by the responder will be searched for the signer certificate, unless the
-// |OCSP_NOINTERN| flag is set.
+// OCSP_NOINTERN is for |OCSP_basic_verify| and |OCSP_request_verify|.
+// Certificates included within |bs| or |req| will be included in the
+// search for the signing certificate by default, unless |OCSP_NOINTERN| is set.
 #define OCSP_NOINTERN 0x2
-// OCSP_NOCHAIN is for |OCSP_basic_verify|. All certificates in |certs| and in
-// |bs| are considered as untrusted certificates for the construction of the
-// validation path for the signer certificate unless the |OCSP_NOCHAIN| flag is
-// set.
+// OCSP_NOCHAIN is for |OCSP_basic_verify| and |OCSP_request_verify|. All
+// certificates in both |certs| and in |bs|/|req| are considered as untrusted
+// certificates for the construction of the validation path for the signer
+// certificate by default, unless |OCSP_NOCHAIN| is set.
```

Comment:
> unless |OCSP_NOCHAIN| is set

Just checking: the negation implied by "unless" means that the `certs` and all "intern" certs in `bs/req` are not added for the purpose of chain building. It doesn't mean negating "untrusted" to "trusted.

Reply:
Yeah this could be clearer. `OCSP_NOCHAIN` means that certificates in `certs` and `bs/req` aren't considered when building the chain.
But on closer examination, `certs` is never considered in `OCSP_request_verify` so the above only applies to `OCSP_basic_verify`. Will change.